src/otx/recipe/_base_/data/classification.yaml

```diff
@@ -18,21 +19,31 @@ train_subset:
   transforms:
     - class_path: otx.core.data.transform_libs.torchvision.RandomResizedCrop
       init_args:
-        scale: $(input_size)
+        scale: [256, 256]  # Adjusted scale
     - class_path: otx.core.data.transform_libs.torchvision.PhotoMetricDistortion
-      enable: false
+      enable: true  # Enabled color distortions
     - class_path: otx.core.data.transform_libs.torchvision.RandomAffine
-      enable: false
+      enable: true  # Enabled affine transformations
     - class_path: otx.core.data.transform_libs.torchvision.RandomFlip
       init_args:
         prob: 0.5
         is_numpy_to_tvtensor: true
     - class_path: torchvision.transforms.v2.RandomVerticalFlip
-      enable: false
+      enable: true  # Enabled vertical flipping
     - class_path: torchvision.transforms.v2.GaussianBlur
-      enable: false
+      enable: true
+      init_args:
+        kernel_size: 3  # Reduced blur intensity
+    - class_path: torchvision.transforms.v2.GaussianNoise
```

Comment:
It seems like a lot of augmentations for me. I think there should be an ablation study if we really need all of them.